src/common/paths/write.yml

```diff
@@ -131,8 +137,16 @@ post:
                     <center>nginx</center>
                   </body>
                 </html>
+        application/json:
```

Comment:
mulitple content types is technically correct here, though unfortunately the openapi-generator codegen tool does not handle it very well. But, it handles it well enough for the CLI.

Reply:
In practice, the codegen just has `application/json` take priority, and the schema is `string`, so it supports both well enough for what we need - ugly but it works.